repro.in

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
```suggestion
#!/usr/bin/bash
```

Do the same with `buildinfo` please.

Reply:
I understand that using the same shebang in both script makes sense.
But why `#!/usr/bin/bash` over `#!/bin/bash`?
Some other scripts like `makepkg` or `makerepropkg` are also using `#!/bin/bash`

Reply:
I'm not sure about Debian's usrmerge status, they ship their bash at `/bin/bash` and `/usr/bin/bash` is not necessarily available, however on usrmerge systems like Arch Linux `/bin` is symlinked to `/usr/bin`.